packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/repository.security_extension.test.ts

```diff
@@ -213,71 +218,57 @@ describe('SavedObjectsRepository Security Extension', () => {
         })
       );
     });
-
-    test(`adds audit event when successful`, async () => {
```

Comment:
We can no longer validate audit logging for SO's at the repository level because all auditing is now contained within the security extension. Testing audit logging can be found in saved_objects_security_extension.test.ts